src/main/resources/citizen/templates/document/defendantResponseReceipt.html

```diff
@@ -338,40 +338,7 @@
         </td>
       </tr>
       {% endif %}
-      {% if defendant.events is defined %}
```

Comment:
Removed this block as it is not required. Luckily the conditions  `{% if defendant.events is defined %` and `{% if defendant.evidences is defined %}` were never true, so we were not seeing duplicate on pdf.